lib/Controller/RoomController.php

```diff
@@ -731,27 +726,19 @@ protected function createCircleRoom(string $targetCircleId): DataResponse {
 	 * @return DataResponse
 	 */
 	protected function createEmptyRoom(string $roomName, bool $public = true): DataResponse {
-		$roomName = trim($roomName);
-		if ($roomName === '') {
-			return new DataResponse([], Http::STATUS_BAD_REQUEST);
-		}
-
 		$currentUser = $this->userManager->get($this->userId);
-
 		if (!$currentUser instanceof IUser) {
 			return new DataResponse([], Http::STATUS_NOT_FOUND);
 		}
 
+		$roomType = $public ? Room::PUBLIC_CALL : Room::GROUP_CALL;
+
 		// Create the room
-		if ($public) {
-			$room = $this->manager->createPublicRoom($roomName);
-		} else {
-			$room = $this->manager->createGroupRoom($roomName);
+		try {
+			$room = $this->roomService->createConversation($roomType, $roomName, $currentUser);
+		} catch (\InvalidArgumentException $e) {
+			return new DataResponse([], Http::STATUS_BAD_REQUEST);
```

Comment:
Should we expose what field is at fault in the response?

Reply:
I'm not sure it's of a lot of help atm, the only invalid input is the name, as object data can not be specified via API and the type is hard coded in the controller.